src/content/app-platform/create-catalog/index.md

```diff
@@ -0,0 +1,164 @@
+
+## Publishing an app to the app catalog
+
+Now you can configure your apps to be published to your catalog. We have created
+a GitHub Action for `app-build-suite` that will automate this.
+
+```yaml
+# .github/workflows/push-to-app-catalog.yaml
+name: 'Push to App Catalog'
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  push_to_app_catalog:
+    uses: giantswarm/app-build-suite/.github/workflows/push-to-app-catalog.yaml@v1.1.0
+    with:
+      app_catalog: example-catalog
+      chart: hello-world-app
+      organization: example-org
+    secrets:
+      envPAT: ${{ secrets.PAT }}
+```
+
+To configure the action you need to add a secret called `PAT` to the app's git
+repository. This must contain a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+that has write permission to the app catalog git repository.
+
+We recommend using [Dependabot](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot)
+to keep the GitHub Action up to date.
+
+## Create Catalog CR
+
+Once you've created your app catalog and published some apps you need to create
+a Catalog CR in the management cluster to register it with app platform. This
+can be done using [kubectl gs template catalog]({{< relref "/ui-api/kubectl-gs/template-catalog" >}}).
+
+You should create the Catalog CR in the [organization]({{< relref "/general/organizations" >}})
+namespace where the apps will be used.
+
+```nohighlight
+kubectl gs template catalog \
+  --name example \
+  --namespace org-example \
+  --description "An example Catalog" \
+  --url https://example.github.io/example-catalog/ \
+  --logo https://example.com/logos/example-logo.png
+```
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: Catalog
+metadata:
+  creationTimestamp: null
+  name: example
+  namespace: org-example
+spec:
+  description: An example Catalog
+  logoURL: https://example.com/logos/example-logo.png
+  storage:
+    URL: https://example.github.io/example-catalog/
+    type: helm
+  title: example
+```
+
+## Referencing an app catalog in an App CR
+
+Since the Catalog CR is not in the `default` namespace you need to set the catalog
+namespace to the organization namespace where it is stored. This can be done using
+[kubectl gs template app]({{< relref "/ui-api/kubectl-gs/template-app" >}}).
+
+```nohighlight
+kubectl gs template app \
+  --catalog example \
+  --catalog-namespace org-example \
+  --name hello-world-app \
+  --namespace default \
+  --cluster 2hr7z  \
+  --version 0.1.0
+```
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: example-app
+  namespace: 2hr7z
+spec:
+  catalog: example
+  catalogNamespace: org-example
+  kubeConfig:
+    inCluster: false
+  name: example-app
+  namespace: default
+  version: 0.1.0
+```
+
+## Viewing app metadata
```

Comment:
I would call this something like "Browsing the catalog" or "Inspecting the catalog"

Reply:
Thanks I like inspecting for this.